README.md

```diff
@@ -13,13 +13,15 @@ There are two ways of using this repository:
 
 * **Platform & Application Layer**
 
-  Configure credentials for infrastructure providers such as AWS, Azure, GCP
+  Configure credentials for infrastructure providers such as AWS, Azure, GCP, digitalocean,
   and/or Packet, then execute the Terraform process to automatically deploy the
   required machines ([Platform Layer](#platform-layer)) and setup the
   [Application Layer](#application-layer).
 
   See the [Complete Guide](GUIDE_COMPLETE.md) for more.
 
+  See the [Digital Ocean Guide](DIGITAL_OCEAN_README.md) for some gotchas.
```

Comment:
As mentioned earlier, these are not DO specific changes, so I'd remove changes to this file.

Reply:
done.

Reply:
Done.